tests/StorageBackendTest.php

```diff
@@ -115,6 +115,31 @@ public function storageBackendOptionsProvider(): \Generator
         ];
     }
 
+    /**
+     * @dataProvider storageBackendOptionsProvider
+     */
+    public function testUpdateStorageBackend(array $options)
```

Comment:
nevim jestli by nebylo vhodne ten test rozdelit na "validni" a "nevalidni" scenario. Imho by to principialne tak melo byt a bude to prehlednejsi. Ale ztratime kousicek vykonu. Necham na tobe

Reply:
rozdelil som to